### DISCUSSION TITLE
Add plugin menu to existing top level navigation menu item

### DISCUSSION THREAD

**Discussion by danielxt** [netbox-community/netbox#16130] [Help Wanted!] [answered]
#9071 proposed 2 features to be added to plugin navigation visibility:

1. Allow plugins to register their own top level navigation section in the navigation menu
2. Allow the order of top level navigation sections to be influenced with a "weight" attribute

In the following discussion, a third feature was requested:

3. Add ability to add plugin menu to existing top level navigation sections like "Virtualization" or "Devices"

The fix #10441 was later merged, but from what I can see and the [navigation documentation](https://github.com/netbox-community/netbox/blob/develop/docs/plugins/development/navigation.md), only feature 1 was implemented. 

Is feature 2 and 3 possible with Netbox currently?

**Comment by abhi1693** [comment, marked as answer]
Not possible as of writing this

**Comment by jeremystretch** [comment]
We don't permit plugins to override or extend the core menus because it can lead to conflicts and confusion for the user, especially as we add and change menu items over time.
